### DISCUSSION TITLE
What's the meaning of "monorepo" in the renovate context?

### DISCUSSION THREAD

**Discussion by martinbonnin** [renovatebot/renovate#21123] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

We just got [this PR](https://github.com/joreilly/Confetti/pull/396) open whose titles says: `Update kotlin monorepo to v1.8.20-RC2`. The "monorepo" term sounds weird in this context. I understand that several dependencies are released together in [the Kotlin monorepo ](https://github.com/JetBrains/kotlin) but from a consumer point of view, all I want is that a version is updated: 

```yaml
-kotlin = "1.8.20-RC"
+kotlin = "1.8.20-RC2"
```

Could the title be changed to 

```
Update version 'kotlin' to 
 1.8.20-RC2
```

Like there is `Update dependency 'foo' to barVersion`

?

**Comment by rarkins** [comment]
The term "monorepo" is used when multiple packages from the same repo are grouped together.

If there is only one package in the PR, you will see that package in the PR title instead. But in the example you linked to, four different packages are controlled by that variable so the group name ("kotlin monorepo") is used.

> Could the title be changed to

Renovate is highly flexible so I'm sure you could figure out a way to achieve what you want through packageRules config, but it's up to you how much time you think it's worth if it's only a cosmetic change.

> Like there is Update dependency 'foo' to barVersion

Yes, that's how it normally is when only one dependency is updated. Not when 2+ are

**Reply by martinbonnin** [reply]
Thanks for the quick reply!

**Reply by rarkins** [reply]
BTW the rule which creates this is here: https://docs.renovatebot.com/presets-group/#groupkotlinmonorepo
